lua/blink/cmp/lib/text_edits.lua

```diff
@@ -142,6 +142,18 @@ function text_edits.get_from_item(item)
   local offset_encoding = text_edits.offset_encoding_from_item(item)
   text_edit = text_edits.to_utf_8(text_edit, offset_encoding)
 
+  local mode = context.get_mode()
+  if mode == 'cmdline' then
+    local line = context.get_line()
+    -- Detect range markers '<,'>' or numeric ranges '3,5'
+    local prefix = line:match("^%s*['<,>%d]+%s*")
```

Comment:
following is more accurate

```suggestion
    local prefix = line:match("^%s*'<%s*,%s*'>%s*") or line:match("^%s*%d%s*,%s*%d%s*")
```

Reply:
Tthanks, I moved the regex matching elsewhere as suggested by @Saghen.